app/src/features/surveys/animals/profile/captures/components/capture-card-details/components/ReleaseDetails.tsx

```diff
@@ -39,7 +39,7 @@ export const ReleaseDetails = (props: IReleaseDetailsProps) => {
               Release time
             </Typography>
             <Typography color="textSecondary" variant="body2">
-              {getFormattedDate(DATE_FORMAT.MediumDateTimeFormat, releaseTimestamp)}
+              {getFormattedDate(DATE_FORMAT.MediumDateTimeFormat, releaseDate)}
```

Comment:
I think this needs to first combine the captureDate and captureTime. Otherwise the UI always shows `12:00 am`.